### PR TITLE
@craigspaeth => Content Differentiation for Team Blogs

### PR DESCRIPTION
**Pull request by kanaabe**
Closes https://github.com/artsy/force/issues/41 https://github.com/artsy/force/issues/42 https://github.com/artsy/force/issues/44 https://github.com/artsy/force/issues/46 https://github.com/artsy/force/issues/45

A couple things to note:
- We still support Sections by calling `routes.sections` when a channel isn't found. `routes.sections` finally calls `next()` if a section isn't found either. I chose this way because we'll remove them once we figure out what to do with Sections that don't also exist as a channel.
- Makes `topParselyArticles` more flexible + moves it to the `util` component

### CODE REVIEW

File: apps/articles/client/team_channel.coffee [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ArticlesGridView = require '../../../components/articles_grid/view.coffee'
+initCarousel = require '../../../components/merry_go_round/horizontal_nav_mgr.coffee'
+sd = require('sharify').data
+
+module.exports.TeamChannelView = class TeamChannelView extends Backbone.View
+
+  initialize: ->
+    @channel = new Channel sd.CHANNEL
+    @gridArticles = new Articles
+    @gridArticles.url = "#{@gridArticles.url}/?&published=true&limit=12&sort=-published_at&channel_id=#{@channel.get('id')}"
```
Comment (craigspaeth) [inline apps/articles/client/team_channel.coffee]: You can pass these query params into a fetch call like `gridArticles.fetch(data: { published: true, limit: 12 ... })`. That, or pulling in the querystring module could help clean this up. Not a blocker by any means though.

Comment (kanaabe) [inline apps/articles/client/team_channel.coffee]: Ya true, this module has a More Articles button so params would have to be persisted to subsequent fetches, i'll add a params `option`

File: apps/articles/index.coffee [hunk 2]
```diff
@@ -5,16 +5,18 @@
...
 routes = require './routes'
 { resize, crop } = require '../../components/resizer'
+{ toSentence } = require 'underscore.string'
 
 app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 app.locals.resize = resize
 app.locals.crop = crop
+app.locals.toSentence = toSentence
```
Comment (craigspaeth) [inline apps/articles/index.coffee]: 👌

File: apps/articles/routes.coffee [hunk 3]
```diff
@@ -63,6 +65,34 @@ setupEmailSubscriptions = (user, article, cb) ->
           res.locals.sd.SECTION = section.toJSON()
           res.render 'section', section: section, articles: articles
 
+@teamChannel = (req, res, next) =>
+  new Channel(id: req.params.slug).fetch
```
Comment (craigspaeth) [inline apps/articles/routes.coffee]: Might help clean things up to extract this down to line ~90 to the model layer, maybe like `channel.fetchWithArticles`.

File: apps/articles/test/client/team_channel.coffee [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+{ resolve } = require 'path'
+fixtures = require '../../../../test/helpers/fixtures'
+{ fabricate } = require 'antigravity'
+Articles = require '../../../../collections/articles.coffee'
+Article = require '../../../../models/article.coffee'
+Channel = require '../../../../models/channel.coffee'
+sd = require('sharify').data
+{ resize, crop } = require '../../../../components/resizer'
+
+describe 'TeamChannelView', ->
```
Comment (craigspaeth) [inline apps/articles/test/client/team_channel.coffee]: 👍

File: components/util/parsely.coffee [hunk 5]
```diff
@@ -0,0 +1,20 @@
+request = require 'superagent'
+{ reject } = require 'underscore'
+
+module.exports =
+
+  topParselyArticles: (section, articleSlug, key, secret, cb) ->
```
Comment (craigspaeth) [inline components/util/parsely.coffee]: This seems small enough to just be a method to add to the Article or Articles model/collection (even though it doesn't return _that_ model data)—that might also help DRY up some of that router fetching I noted above by co-locating that code.

**Comment by craigspaeth** [comment]
👏  Nice! I have some non-blocking comments. Feel free to address at your leisure and merge when ready. Good job!

File: components/util/parsely.coffee (hunk 5 above)
Comment (kanaabe) [inline components/util/parsely.coffee]: The move to `util` was influenced by the likelihood that 1. we'll use more parsely fetching in homepage/tag pages and 2. running the class-level `Article.topParselyArticles` in the `Articles` subapp just seemed a bit off to me.

Comment (craigspaeth) [inline components/util/parsely.coffee]: 👌 I support that reasoning. Feel free to leave in components/util then :)

**Pull request merged**